src/API/Management/Jobs.php

```diff
@@ -35,15 +35,19 @@ public function getErrors($id)
      *
      * @param  string $file_path
      * @param  string $connection_id
+     * @param  boolean $upsert
+     * @param  boolean $send_completion_email
      * @return mixed
      */
-    public function importUsers($file_path, $connection_id)
+    public function importUsers($file_path, $connection_id, $upsert = false, $send_completion_email = true)
```

Comment:
We've got another optional parameter we can send, `external_id`. At this point, I think it would be better to pass those optional params in as an array to disambiguate the call and provide a way to add new params if that endpoints begins supporting more.

Reply:
I've committed the change. Can you see if it works?